homeassistant/components/asuswrt/router.py

```diff
@@ -364,14 +416,14 @@ def host(self) -> str:
         return self._api.host
 
     @property
-    def unique_id(self) -> str | None:
-        """Return router unique id."""
-        return self._entry.unique_id
+    def mac(self) -> str | None:
+        """Return router mac address."""
+        return self._api.label_mac
 
     @property
-    def name(self) -> str:
-        """Return router name."""
-        return self.host if self.unique_id else DEFAULT_NAME
+    def unique_id(self) -> str | None:
```

Comment:
This can already be set in the constructor right? We don't need a property method for this?

Reply:
Property is used by sensors to generate `unique_id`. Do you mead that I should set the value in an attribute and return here the attribute?